dev/tasks/verify-rc/github.nix.yml

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# NOTE: must set "Crossbow" as name to have the badge links working in the
+# github comment reports!
+name: Crossbow
+
+on:
+  push:
+    branches:
+      - "*-github-*"
+
+jobs:
+  verify:
+    name: "Verify release candidate {{ os }} {{ artifact }} {{ flag }}"
+    runs-on: {{ os }}-latest
+    steps:
+      - name: Checkout Arrow
+        run: |
+          git clone --no-checkout {{ arrow.remote }} arrow
+          git -C arrow fetch -t {{ arrow.remote }} {{ arrow.branch }}
+          git -C arrow checkout FETCH_HEAD
+          git -C arrow submodule update --init --recursive
+      - name: Fetch Submodules and Tags
+        shell: bash
+        run: cd arrow && ci/scripts/util_checkout.sh
+      - name: Run verification
+        shell: bash
+        run: |
+          set -e
+
+          {{ flag }}
+          if [ $(uname) = "Darwin" ]; then
+            brew install llvm@7 thrift glib
+          else
+            sudo apt-get --purge remove -y llvm-8 llvm-9
+            sudo apt-get install -y \
+              llvm-7 \
+              wget curl libboost-all-dev jq \
+              autoconf-archive gtk-doc-tools libgirepository1.0-dev flex bison
+            if [ "$TEST_JAVA" = "1" ]; then
+              # Maven
+              MAVEN_VERSION=3.6.3
+              wget http://apache.mirrors.lucidnetworks.net/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.zip
+              unzip apache-maven-$MAVEN_VERSION-bin.zip
+              mkdir -p $HOME/java
+              mv apache-maven-$MAVEN_VERSION $HOME/java
+              export PATH=$HOME/java/apache-maven-$MAVEN_VERSION/bin:$PATH
+            fi
+            if [ "$TEST_RUBY" = "1" ]; then
+              ruby --version
+            fi
+          fi
+          # TODO: put 0.16.0 2 in some separate file?
+          # If you edit the versions, be sure to edit the other workflow files in this directory too
+          TEST_DEFAULT=0 {{ flag }} arrow/dev/release/verify-release-candidate.sh {{ artifact }} 0.16.0 2
```

Comment:
Is there a way to provide template parameters in the Crossbow invocation?

Reply:
Not that I know of but you may know better